sphinx_gallery/scrapers.py

```diff
@@ -455,13 +453,14 @@ def figure_rst(figure_list, sources_dir, fig_titles="", srcsetpaths=None):
 
 
 def _get_srcset_st(sources_dir, hinames):
-    """
-    Create the srcset string for including on the rst line.
-    ie. sources_dir might be /home/sample-proj/source,
-    hinames posix paths to
+    """Create the srcset string for including on the rst line.
+
+    For example; `sources_dir` might be `/home/sample-proj/source`,
+    hinames posix paths:
     0: /home/sample-proj/source/plot_types/images/img1.png,
     2.0: /home/sample-proj/source/plot_types/images/img1_2_00x.png,
-    The result will be:
+
+    The result wwould be:
```

Comment:
```suggestion
    The result would be:
```
Also there are spelling failures in CIs